docs/source/science/GEDI/GEDI_L4B.ipynb

```diff
@@ -2,33 +2,80 @@
  "cells": [
```

Comment:
<p>Section title?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Also should be normal text😅</p>